dolphin/db/sqlalchemy/api.py

```diff
@@ -136,23 +150,37 @@ def storage_get(context, storage_id):
 
 def storage_get_all(context, marker=None, limit=None, sort_keys=None,
                     sort_dirs=None, filters=None, offset=None):
-    """Retrieves all storage devices."""
-
     this_session = get_session()
     this_session.begin()
-    # TODO: need to handle all input parameters
-    all_storages = this_session.query(Storage).all()
-    return all_storages
-
-
-def volume_create(context, values):
-    """Create a volume from the values dictionary."""
-    return NotImplemented
-
+    if not sort_keys:
+        sort_keys= ['created_at']
```

Comment:
This file may also need to be formatted.

Reply:
done